Simple solvers/SB.py

```diff
@@ -12,17 +12,17 @@
 b = float(input("What deos B equal to?"))
 
 c = float(input("What does C equal to?"))
-dis = b**2-4*a*c
+discriminant = b**2-4*a*c
 
 
-if dis < 0:
+if discriminant < 0:
```

Comment:
Try adding a comment before each block of code to tell the reader what the code is supposed to do.